x-pack/plugins/ml/server/routes/job_service.ts

```diff
@@ -285,6 +285,44 @@ export function jobServiceRoutes({ router, routeGuard }: RouteInitialization) {
     })
   );
 
+  /**
+   * @apiGroup JobService
+   *
+   * @api {post} /api/ml/jobs/jobs_with_geo Jobs summary
+   * @apiName JobsSummary
+   * @apiDescription Returns a list of anomaly detection jobs with analysis config with fields supported by maps.
+   *
+   * @apiSchema (body) optionalJobIdsSchema
+   *
+   * @apiSuccess {Array} jobIds list of job ids.
+   */
+  router.get(
+    {
+      path: '/api/ml/jobs/jobs_with_geo',
+      validate: false,
```

Comment:
You've mentioned `optionalJobIdsSchema` for api docs but it's not a part of the `validate` definition.

Reply:
Removed in c88762c614ab8c03be504dbde24242d5bacf1107